cudapoa/README.md

```diff
@@ -0,0 +1,23 @@
+# CUDAPOA
+
+The `cudapoa` package provides a GPU-accelerated implementation of the [Partial Order Alignment](https://simpsonlab.github.io/2015/05/01/understanding-poa/)
+algorithm. It is heavily influenced by [SPOA](https://github.com/rvaser/spoa) and in many cases can be considered a GPU-accelerated replacement. Features include:
+
+## Tool
+
+A command line tool for generating consensus and MSA from a list of `fasta`/`fastq` files. The tool
+is built on top of `libcudapoa.so` and showcases optimization strategies for writing high performance
+applications with `libcudapoa.so`.
+
+## Library
+Built as `libcudapoa.[so|a]`
+
+* Generation of consensus sequences
```

Comment:
I think these points are supposed to be above the "Tool" section.

Reply:
They're meant to be part of the library section, but I reorganized and moved the library component to the top.

Reply:
I was confused by the features include part in the cudapoa description too. Can you please remove that opening?

Reply:
totally missed that. Done